clientv3/logger.go

```diff
@@ -35,19 +38,31 @@ type settableLogger struct {
 
 func init() {
 	// use go's standard logger by default like grpc
-	logger.mu.Lock()
-	logger.l = log.New(os.Stderr, "", log.LstdFlags)
-	grpclog.SetLogger(&logger)
-	logger.mu.Unlock()
+	logger.set(log.New(os.Stderr, "", log.LstdFlags))
+
+	// disable client side grpc by default
+	grpclog.SetLogger(log.New(ioutil.Discard, "", 0))
```

Comment:
@heyitsanthony You are right, I forgot about that.

Just fixed and adds more comments with your feedback.

PTAL.

Thanks!